src/types.ts

```diff
@@ -78,7 +78,7 @@ export type IdentityResultionResult = object
 export interface ExternalCallHandler {
     ajaxGet?: (
         url: string,
-        onSuccess: (responseText: string) => void,
+        onSuccess: (responseText: string, response: any) => void,
```

Comment:
```suggestion
        onSuccess: (responseText: string, response?: any) => void,
```

Reply:
we actually always have it

Reply:
oh ok because I saw that previously it wasn't there

Reply:
yeah, it was missing in the types